docs/book/src/migration/multi-group.md

```diff
@@ -4,32 +4,35 @@
 
 <h1>Note</h1>
 
-Multi-group scaffolding support was not present in the initial version of
-the Kubebuilder v2 scaffolding (as of Kubebuilder v2.0.0).
-
-To change the layout of your project to support Multi-Group run the command
-`kubebuilder edit --multigroup=true`. Once you switch to a multi-group layout, the new Kinds
-will be generated in the new layout but additional manual work is needed 
-to move the old API groups to the new layout.
-
-</aside>
-
-While Kubebuilder v2 will not scaffold out a project structure compatible
+While Kubebuilder will not scaffold out a project structure compatible
 with multiple API groups in the same repository by default, it's possible
 to modify the default project structure to support it.
 
+</aside>
+
 Let's migrate the [CronJob example][cronjob-tutorial].
 
 <aside class="note warning">
-<h1>Using go/v4</h1>
+<h1>Instructions vary per project layout</h1>
+
+You can verify the version by looking at the PROJECT file. The currently default and
+recommended version is go/v4.
 
-If you create your project using go/v4 plugin (you can verify it by looking at the PROJECT file )
-then, all steps are the same but you need to keep in mind that the api and controllers directory
-path is now under the `pkg` directory instead. So, you need ensure that you update the
-paths accordingly.
+The layout go/v3 is **deprecated**, if you are using go/v3 it is recommended that you migrate to
+go/v4, however this documentation is still valid. [Migration from go/v3 to go/v4][migration-guide].
+
+This document is incompatible with go/v2. The documentation for go/v2 is still available in
+the [legacy documentation](https://book-v2.book.kubebuilder.io/migration/multi-group.html).
+
+The layout go/v1 is incompatible with multi-group.
```

Comment:
```suggestion
```

It is very unlike someone been using v1/v2 if so they should check the old docs and not this one,
Anyway, if that does not clarify does we can address via chart or issues.

However, here I think we could try to make it more compreensive and help 90% of the audience.